integration/controller_test.go

```diff
@@ -267,6 +267,28 @@ var _ = Describe("create", func() {
 				}, Timeout, PollingInterval).Should(WithTransform(getFirstOwnerName, Equal(ss.GetName())))
 			})
 		})
+
+		Context("Secret recreation", func() {
+			BeforeEach(func() {
```

Comment:
Not sure about this. I think this is being executed before the SealedSecret is created?
The SealedSecret is created in a `JustBeforeEach` function.

Reply:
let me check, you are right. Thanks ^^